packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -364,7 +364,8 @@ export enum PersonaSize {
   size40 = 12,
   size48 = 13,
   size72 = 14,
-  size100 = 15
+  size100 = 15,
+  size120 = 16
```

Comment:
I have forwarded this PR to the designer that owns the persona controls (Matt) for him to take a look

---
In reply to: [314568783](https://github.com/OfficeDev/office-ui-fabric-react/pull/10162#discussion_r314568783) [](ancestors = 314568783)

Reply:
The addition of the larger size gives me the appropriate size for use in the Yammer Group Header, which houses a 120px * 120px avatar. I'd created a size mapping in Yammer, but it became clear that it was redundant with the existence of PersonaSize (especially since we're now using a Persona component for the avatar). So, adding another size to the PersonaSize enum seemed to make the most sense. I'm updating the styles that you mentioned as well and will be pushing an update shortly.

Reply:
@Vitalius1 , Okay. I've updated the styles, added to the Persona story, and examples. However, when I run: `yarn build` I'm seeing a failure in a test that I don't fully understand (because I don't believe that I touched the files under test, and the scenario looks identical to the surrounding ones in the same test file). I'm re-running the command so I can add a screenshot here as well.

Reply:
It is possible because you added an example you needed to update the snapshots with `yarn update-snapshots` command at the root of the repo.

Reply:
Yeah, I did that as well as added the change file. Not sure which thing fixed it, but something appears to have fixed that test. Now my only blocker is the Screener tests. I don't have a login for Screener for some reason. I do have a MSFT-based login that works for Screener for the yammer-clients project, though.

Reply:
Screener can only be approved by a few people with permissions from Fabric team. Main person who approves any screener changes is @betrue-final-final , but it looks like he approved the screener changes.